frontend/src/component/Dashboard/AgendaInfo.styl

```diff
@@ -3,26 +3,26 @@
 .agendaInfo
   &__header
     font-weight bold
-    margin-bottom 10px
-    margin-top 10px
+    margin-bottom standardSpacing
+    margin-top standardSpacing
   &__content
     &__text
-      margin-bottom 15px
+      margin-bottom standardSpacing
       &__help
         margin-left 5px
     &__link
       display flex
       align-items center
       &__icon
-        padding 5px 15px
+        padding 5px standardSpacing
         border-top-left-radius 5px
         border-bottom-left-radius 5px
```

Comment:
borderRadiusStandard

Reply:
Same that above. If it's ok for you, I'll update this in the next PR, to be able to merge this one asap.